modules/Components/src/client/ACASThingBrowserView.html

```diff
@@ -71,14 +71,15 @@ <h5>Searching for <%-thingName%>s containing text "<i class="bv_exptSearchTerm">
                                 <div class="bar" style="width: 100%; margin: auto;"></div>
                             </div>
                         </div>
+                        <h5 class="bv_maxThingBrowserSearchResultsReached hide"></h5>
                     </td>
                 </tr>
                 </tbody>
                 </tbody>
             </table>
         </div>
-
         <div class="bv_thingTableController hide" style="margin:0px; padding:10px; overflow: auto; overflow: auto;"></div>
+        <label class="bv_maxThingBrowserSearchResultsReached" style="margin-left:10px;"></label>
```

Comment:
Is this same `bv_maxThingBrowserSearchResultsReached` intended to be in here twice?

Reply:
Thanks, removed the extra one